.changeset/three-emus-tell.md

```diff
@@ -0,0 +1,116 @@
+---
+"@salt-ds/lab": patch
+---
+
+- `DatePickerSinglePanel` has been renamed to `DatePickerSingleGridPanel`. This component now provides up to 12 `Calendars` within a panel via the `numberOfVisibleMonths` and `columns` props, enabling multi-month date selection (defaults to 1 visible calendar).
+
+```diff
+<DatePicker
+  selectionVariant="single"
+>
+  <DatePickerTrigger>
+    <DatePickerSingleInput />
+  </DatePickerTrigger>
+  <DatePickerOverlay>
+    <DatePickerSingleGridPanel numberOfVisibleMonths={12} columns={3} />
+  </DatePickerOverlay>
+</DatePicker>
+```
+
+- `DatePickerRangeGridPanel` has been added. It provides up to 12 Calendars within a panel via the `numberOfVisibleMonths` and `columns` props, enabling multi-month date range selection (defaults to 2 visible calendars and 2 columns).
+
+```
+<DatePicker
+  selectionVariant="range"
+>
+  <DatePickerTrigger>
+    <DatePickerRangeInput />
+  </DatePickerTrigger>
+  <DatePickerOverlay>
+    <DatePickerRangeGridPanel numberOfVisibleMonths={12} columns={3} />
+  </DatePickerOverlay>
+</DatePicker>
+```
+
+Note: `DatePickerRangePanel` remains unchanged and provides a dual `Calendar` date view, with individual month/year dropdowns for the start and end date range.
+
+- `CalendarWeekHeader` is now rendered by `CalendarGrid` and can be customized using the `CalendarWeekHeaderProps` prop.
```

Comment:
Is it better to add what a user should do with this. e.g., most likely remove `<CalendarWeekHeader />` if in use. I only get this when looking at diff of the example code below.

Reply:
done